src/Heading.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./App.css";
+
+const Heading = () => {
+    return(
+        <header className="App-header">
+            <img alt="App-logo" className="App-logo" src="https://marketplace.canva.com/EAE8Xyb2xY8/1/0/1600w/canva-gold-exclusive-royal-luxury-hotel-logo-izeElzvImEY.jpg"></img>
+            <h1>CYF Hotel</h1></header>
```

Comment:
It's a minor point, but just keep an eye on the formatting of your embedded html.
It would be good `</header>` was on a new line